.github/workflows/android.yml

```diff
@@ -31,7 +31,9 @@ jobs:
       - name: Cache dependencies
         uses: actions/cache@v2.1.3
         with:
-          path: 3rdParty/buildCache
+          path: |
+            3rdParty/buildCache
+            !3rdParty/buildCache/android/vcpkgcache/vcpkg/archives/
```

Comment:
Please change it to `!3rdParty/buildCache/android/vcpkgcache/`

Reply:
Fixed